### ISSUE TITLE
ArrayIndex Out of Bounds Exception

### ISSUE THREAD

**Issue by NicoKiaru** [saalfeldlab/bigwarp#127]
Hi @bogovicj, 

I'm using BigWarp to build a 'ground truth` transform in 2D with around 3000 lanmarks.

Quite frequently, I find this kind of error, when I clicked to activate or deactivate some landmarks

```
load landmarks
Exception in thread "Thread-13" java.lang.ArrayIndexOutOfBoundsException: 8
	at bigwarp.landmarks.LandmarkTableModel.copyLandmarks(LandmarkTableModel.java:1203)
	at bigwarp.transforms.TpsTransformSolver.solve(TpsTransformSolver.java:37)
	at bigwarp.transforms.TpsTransformSolver.solve(TpsTransformSolver.java:23)
	at bigwarp.transforms.BigWarpTransform.getTransformation(BigWarpTransform.java:73)
	at bigwarp.BigWarp$SolveThread.run(BigWarp.java:2968)
```

**Comment by bogovicj** [comment]
Thanks @NicoKiaru , I'm able to reproduce this.  Checking it out.

**Comment by bogovicj** [comment]
I think this is fixed in this branch
https://github.com/saalfeldlab/bigwarp/tree/ltmSynch
but will test a little more before merging

**Comment by bogovicj** [comment]
This should be solved after #129 , thanks again!
